chaoscenter/graphql/server/Dockerfile

```diff
@@ -11,21 +11,36 @@ WORKDIR /gql-server
 
 ENV GOOS=${TARGETOS} \
     GOARCH=${TARGETARCH}
-    
+
 RUN go env
 RUN CGO_ENABLED=0 go build -o /output/server -v
 
 ## DEPLOY STAGE
-# Image source: https://github.com/litmuschaos/test-tools/blob/master/custom/hardened-alpine/infra/Dockerfile
-# The base image is non-root (have litmus user) with default litmus directory.
-FROM litmuschaos/infra-alpine
+# Use Red Hat UBI minimal image as base
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.4
 
 LABEL maintainer="LitmusChaos"
 
-COPY --from=builder /output/server /litmus
-# Copy manifests and change the group to 0(root) to make it Openshift friendly
-# as Openshift runs container with an arbitrary uid that in the root group
-COPY --chown=litmus:0 --chmod=750 ./manifests/. /litmus/manifests
+ENV APP_USER=litmus
+ENV APP_DIR="/litmus"
+ENV DATA_DIR "$APP_DIR/data"
+ENV CONF_DIR "$APP_DIR/conf"
+
+RUN microdnf install shadow-utils \
+    && groupadd -g 2000 $APP_USER \
+    && useradd -u 2000 -g 2000 -d $APP_DIR -s /sbin/nologin $APP_USER \
+    && mkdir -p "$DATA_DIR" "$CONF_DIR" \
+    && chown -R "$APP_USER:0" "$DATA_DIR" "$CONF_DIR" \
+    && chmod -R 700 "$DATA_DIR" "$CONF_DIR"
```

Comment:
This we can remove for this image as this will only be required for go-runner , there we would need sudo permissions as non-root users.